src/Stores/SubjectsPromises/reducer.js

```diff
@@ -0,0 +1,33 @@
+/* @flow */
+import type { SubjectPromise, Action } from '../../types';
+
+export type State = {
+  +subjectsPromises: ?Array<SubjectPromise>,
+  +loading: boolean,
+  +error: *,
+};
+
+const initialState = {
+  subjectsPromises: null,
```

Comment:
thoughts on calling this `data`? We are doing that at work.

So instead of doing `subjectsPromises.subjectsPromises` (which is a bit meh), we do `subjectPromises.data`

Reply:
Good call. I'll update